src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt

```diff
@@ -24,22 +25,28 @@ open class HiddenSecretsPlugin : Plugin<Project> {
         private const val KOTLIN_FILE_NAME = "Secrets.kt"
 
         //Tasks
+        const val TASK_GROUP = "Hide secrets"
```

Comment:
CamelCase ?

Reply:
@GhassenMsd no it must stay like this do display the group correctly